cross-chain-atomic-swap-cordapp/build.gradle

```diff
@@ -135,6 +135,9 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
             address "0.0.0.0:10003"
             adminAddress "0.0.0.0:10103"
         }
+        projectCordapp {
+            config project.file("src/main/resources/config.conf")
```

Comment:
I think you can move this line into `nodeDefaults`.`projectCordapp` so that you don't need to write this 4 times.

Reply:
Yep indeed, cleaned :)